RecoParticleFlow/PFProducer/python/particleFlowBlock_cfi.py

```diff
@@ -136,4 +136,52 @@
         )          
 )
 
+def _findIndicesByModule(name):
+   ret = []
+   for i, pset in enumerate(particleFlowBlock.elementImporters):
+        if pset.importerName.value() == name:
+            ret.append(i)
+   return ret
 
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+# kill tracks in the HGCal
+_insertGeneralTracksImporter = {}
+for idx in _findIndicesByModule('GeneralTracksImporter'):
+    _insertGeneralTracksImporter[idx] = dict(
+        importerName = cms.string('GeneralTracksImporterWithVeto'),
+        veto = cms.InputTag('hgcalTrackCollection:TracksInHGCal')
+    )
+phase2_hgcal.toModify(
+    particleFlowBlock,
+    elementImporters = _insertGeneralTracksImporter
```

Comment:
did this move lose ``` linkDefinitions =  ``` ?
The toModify in cff was previously 
```
-phase2_hgcal.toModify(
-    particleFlowBlock,
-    linkDefinitions = _phase2_hgcal_Linkers,
-    elementImporters = _phase2_hgcal_Importers
```

This is in connection with now complete removal of the importers in #17648

Comparisons made in #16450 2023D3Timing  didn't show any reco differences (other than the filled timing variables); so, the linkDefinitions is probably unrelated